src/adapters/autocomplete.js

```diff
@@ -1,12 +1,13 @@
 import Autocomplete from '../vendors/autocomplete'
 import ajax from '../libs/ajax'
-import Poi from '../mapbox/poi'
 import IconManager from '../adapters/icon_manager'
 import nconf from '@qwant/nconf-getter'
 import Store from '../adapters/store'
 import PanelManager from '../proxies/panel_manager'
 import {layout} from '../../config/constants.yml'
 import ExtendedString from "../libs/string";
+import IdunnPoi from "./poi/geocoder_poi";
```

Comment:
Haha, the geocoder API is called "Bragi". Using Idunn, which is the name of poi API is a bit confusing :D
```suggestion
import GeocoderPoi from "./poi/geocoder_poi";
```

Reply:
i have refact & clear confusion